src/clr-icons/element.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { ClarityIconsApi } from './api';
+import { ShapeTemplateObserver } from './utils/shape-template-observer';
+
+export namespace ClarityIcon {
```

Comment:
I'm not happy with the namespace names I gave. Let's change this to `export namespace ElementModule`.